test/unit/runner.spec.js

```diff
@@ -191,7 +195,7 @@ describe('Runner', function() {
 
       global.foo = 'bar';
       global.bar = 'baz';
-      runner.on('fail', function(test, err) {
+      runner.on(EVENT_TEST_FAIL, function(test, err) {
         expect(test.title, 'to be', 'im a test about lions');
         expect(err.message, 'to be', 'global leak detected: bar');
         delete global.foo;
```

Comment:
Why is not deleting `global.bar` _not_ causing issues for tests that follow?

Reply:
I don't know... please make an issue